WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -262,14 +263,14 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleGetUIAElementValue:(FBRouteRequest *)request
 {
-  FBElementCache *elementCache = request.session.elementCache;
-  XCUIElement *element = [elementCache elementForIndex:[request.parameters[@"id"] integerValue]];
-  NSString *value = request.arguments[@"value"];
-  if (!value) {
-    return FBResponseWithErrorMessage(@"Missing value parameter");
-  }
-  [element adjustToPickerWheelValue:value];
-  return FBResponseWithOK();
+    FBElementCache *elementCache = request.session.elementCache;
+    XCUIElement *element = [elementCache elementForIndex:[request.parameters[@"id"] integerValue]];
+    if (element.elementType == XCUIElementTypePickerWheel) {
+        NSString *wheelPickerValue = request.arguments[@"value"];
+        [element adjustToPickerWheelValue:wheelPickerValue];
+        return FBResponseWithOK();
+    }
+    return FBResponseWithErrorMessage([NSString stringWithFormat:@"Element is not of type %@", [FBElementTypeTransformer shortStringWithElementType:XCUIElementTypePickerWheel]]);
```

Comment:
By looking at this I think I'll change `FBResponseWithErrorMessage` to use format instead of string